bool activate
float64 bhLeftSpeed
float64 bhRightSpeed
---
